include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h

```diff
@@ -54,6 +54,15 @@ struct OpBinder {
     return success();
   }
 
+  ParseResult getIdxTensorOperand(Value &valueIdx, int64_t idx) {
```

Comment:
Try to match style of others and name in a way that is self documenting. Recommend `tensorOperandAtIndex`